CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+### citus v10.0 (February 16, 2021) ###
+
+* Adds shard rebalancer
```

Comment:
better wording ?

Reply:
maybe: `Adds community edition of shard rebalancer with `rebalance_table_shards` & `citus_drain_node` UDFs`

Reply:
Maybe two items:
- Adds rebalance_table_shards() to rebalance the shards across the nodes
- Adds citus_drain_node() to move all shards away from any node